<launch>
  <node pkg="polhemus_fastrak_driver" type="fastrak_driver" name="fastrak">
  	<param name="serial"  value="/dev/ttyUSB0"/>
	<param name="base_frame" type="string" value="/polhemus_transmitter_frame"/>
	<param name="current_sensor" type="string" value="/polhemus_receiver_"/>
  </node>
</launch>
